de/manual/ch_Tools_Assistants.xml

```diff
@@ -49,1074 +49,801 @@
+            </para>
+          </listitem>
 
           <listitem>
-            <para>Discard any results that match the previous search results
+            <para>Abgleichen
             </para>
           </listitem>
-        </varlistentry>
 
-        <varlistentry>
-          <term><guilabel>Search only active data</guilabel></term>
+          <listitem>
+            <para>Abschlussbuchungen
+            </para>
+          </listitem>
 
           <listitem>
-            <para>Search only if data are marked as active. This selector is greyed out for the <guilabel>Split
-              Search</guilabel> as it is meaningful only for
-              <link linkend="tool-find-bsnss">business related search</link>
+            <para>Ausgeglichene Buchungen
             </para>
           </listitem>
-        </varlistentry>
-      </variablelist>
 
-      <note>
-        <para>When selecting search criteria keep in mind that criteria are applied to individual splits or fields
-          in transactions, and that the entire transaction is included in the results. There is no
-          visual indication in the results to indicate which split or field met the search criteria.
-        </para>
-      </note>
+          <listitem>
+            <para>Konto
+            </para>
+          </listitem>
+        </itemizedlist>
+        Wenn Sie den Dialog <guilabel>Buchungen suchen</guilabel> in der Registerkarte eines Kontos
+        öffnen, werden nur die zu diesem Konto zugehörigen Buchungsteile durchsucht, es sei denn,
+        Sie wählen als Suchkriterium <guilabel>Alle Konten</guilabel>; siehe die Fußnote dazu in
+        der Tabelle <xref linkend="tool-find-bttns"/> unten. Bei den Buchungsfeldern werden nur die
+        Buchungen berücksichtigt, die mit einem Buchungsteil in diesem Register verknüpft sind,
+        während bei den übrigen Buchungen nur die mit dem Konto verbundenen Buchungsteile
+        durchsucht werden, die diesem Konto zugeordnet sind, außer Sie verwenden die Suchfunktion
+        <guilabel>Alle Konten</guilabel>.
+      </para>
 
-      <para>When search criteria are selected, you can press the <guibutton>Find</guibutton> button. You will be
-        presented with the search results in a new register tab. A report of the search results may
-        created and printed using &mc.rep.acct-rep; . The Account Report has limited formatting
-        options but, like all &app; reports, being in html format, can be copied and pasted into a
-        spreadsheet for further formatting. Alternatively, the Account Report can be exported to an
-        html file, and the html file may be opened in a spreadsheet.
+      <para>Wenn Sie zuvor die Buchungen in einem Register mit &mc.vw.f-by; filtern, dann werden in diesem Konto
+        nur die Buchungen durchsucht, die durch den Filter angezeigt werden. Suchen Sie in einem
+        gefiltertem Kontobuch oder einem <xref linkend="general-journal" />, werden alle
```

Comment:
Ich glaube, ohne Leerzeichen vor / wird der Quelltext angenehmer umgebrochen. Ich glaube weiterhin, ich habe das größtenteils entfernt, aber im Wiki wohl vergessen.

Reply:
Ob das Element mit oder ohne Leerzeichen beendet wird, hat in diesem Fall keine Auswirkung auf die Formatierung bzw. den Zeilenumbruch.
Und die meisten `<xref />` Elemente enthalten, zu mindestens in dieser Datei, das Leerzeichen am Ende.

Reply:
unwichtig